spec/requests/providers/means/restrictions_spec.rb

```diff
@@ -127,12 +127,12 @@
           end
         end
 
-        context "provider checking citizen's answers" do
+        context "provider checking answers" do
           let(:application) { create(:legal_aid_application, :with_applicant, :with_non_passported_state_machine, :checking_non_passported_means) }
 
-          it "redirects to means summary page" do
+          it "redirects to the means summary page" do
```

Comment:
Should we update the description to specify that it is the check capital answers page?